index.js

```diff
@@ -28,3 +28,108 @@ links.forEach((link) => {
+    paragraph: 'A daily selection of privately personalised reads: no accounts or sign-ups required',
+  },
+  {
+    title: 'tonic',
+    image: 'images/orange-crop2.png',
+    paragraph: 'A daily selection of privately personalised reads: no accounts or sign-ups required',
+  },
+  {
+    title: 'multi-post stories',
+    image: 'images/Snapshootportfolio(3).png',
+    paragraph: 'A daily selection of privately personalised reads: no accounts or sign-ups required',
+  },
+];
+
+const projectContainer = document.querySelector('.project-container');
+
+const projectHtml = projects.map((project) => (
+  `
+  <div class="project">
+  <div class="my-img">
+      <img class="mobile-image" src=${project.image} alt="picture identity of a lady">
+  </div>
+  <div class="my-text">
+      <h2>${project.title}</h2>
+      <ol>
+          <li>CANOPY</li>
+          <li><img src="images/Counter.png" alt="counter"></li>
+          <li>Back End Dev</li>
+          <li><img src="images/Counter(2).png" alt="counter"></li>
+          <li>2015</li>
+      </ol>
+      <p>${project.paragraph} </p><br>
+      <ul class="show">
+          <li>html</li>
+          <li>css</li>
+          <li class="ok">Ruby on rails</li>
+          <li>javascrpt</li>
+      </ul>
+      <button class="active button" type="button" id=${projects.indexOf(project)}> See Project</button>
+  </div>
+</div>
+  `
+));
+
+projectContainer.innerHTML = projectHtml;
+
+const projectButtons = document.querySelectorAll('.button');
+
+const popupContainer = document.querySelector('.popup');
+
+projectButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    const currentProject = projects[button.id];
+    popupContainer.innerHTML = `
+    <div class="project1">
+    <img class="popupx" src="images/Iconx-popup.png" alt="an x">
+    <h2>${currentProject.title}</h2>
+    <ul class="list1">
+        <li>CANOPY</li>
+        <li><img src="images/Counter.png" alt="counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="images/Counter(2).png" alt="counter"></li>
+        <li>2015</li>
+    </ul>
+    <div class="my-img1">
+        <img class="mobile-image1" src=${currentProject.image} alt="picture identity of a lady">
+    </div>
+    <div class="my-img2">
+    <img class="mobile-image2" src=${currentProject.image} alt="nature picture">
+    </div>
+    <p>
+    ${currentProject.paragraph}</p>
+    <ul class="show1">
+        <li>html</li>
+        <li>css</li>
+        <li class="ok1">Ruby on rails</li>
+        <li>javascrpt</li>
+    </ul>
+    <div class="house">
+        <div class="button1a">
+            <button type="button" class="button1">See live <img src="images/Icon-popup.png" alt="a clock"></button>
+        </div>
+        <div class=" button1a">
+            <button type="button" class="button2">See Source <img src="images/Vector-popup.png" alt="github">
+            </button>
+        </div>
+    </div>
+</div>
+    `;
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add an overflow of hidden to the body when a Popup Modal is active as that will prevent the user from accidentally scrolling away.

https://user-images.githubusercontent.com/52934128/175764346-db7799be-f49c-4ba3-bf47-faa77504f91f.mp4